cmd/kube-apiserver/app/options/options.go

```diff
@@ -230,6 +231,7 @@ func (s *APIServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.EtcdConfig.CertFile, "etcd-certfile", s.EtcdConfig.CertFile, "SSL certification file used to secure etcd communication")
 	fs.StringVar(&s.EtcdConfig.CAFile, "etcd-cafile", s.EtcdConfig.CAFile, "SSL Certificate Authority file used to secure etcd communication")
 	fs.BoolVar(&s.EtcdConfig.Quorum, "etcd-quorum-read", s.EtcdConfig.Quorum, "If true, enable quorum read")
+	fs.IntVar(&s.EtcdConfig.CacheSize, "etcd-cache-size", s.EtcdConfig.CacheSize, "Number of etcd entries to cache in memory")
```

Comment:
This cache is actually for deserialized objects. It's not got much to do with etcd. "deserialization-cache-size"?